hooks/playbooks/ceph-deploy.yml

```diff
@@ -43,7 +43,13 @@
 
     - name: Extract ceph.conf from secret
       ansible.builtin.set_fact:
-        cifmw_ceph_conf_encoded: "{{ generated_uuid.stdout | from_json | json_query('data.\"ceph.conf\"') }}"
+        cifmw_ceph_conf_encoded: >-
+          {{
+            (
+              generated_uuid.stdout |
+              from_json
+            )['data']['ceph.conf']
```

Comment:
the key has double quotes there, so I guess that it needs to be here too:
`\"ceph.conf\"`

Reply:
podified-multinode-edpm-e2e-nobuild-tagged-crc-testing https://review.rdoproject.org/zuul/build/7310ea58970d4c129e23da44b275a061 : SUCCESS in 1h 06m 16s
and
https://logserver.rdoproject.org/34/51134/2/check/podified-multinode-edpm-e2e-nobuild-tagged-crc-testing/7310ea5/controller/ci-framework-data/artifacts/pre_deploy_ceph_deploy.yml

looks good, thanks for testing.